package.json

```diff
@@ -43,13 +57,15 @@
     "react-router-dom": "5.3.4",
     "react-router-last-location": "2.0.1",
     "reactflow": "^11.1.0",
+    "simple-zustand-devtools": "^1.1.0",
```

Comment:
@thiagoelg was it failing when this lib was a dev dependency? Maybe I can check the environment before trying to load this (which I believe is done at the bottom of the `integrationJsonStore.tsx` file).

Reply:
It failed when being used in web tools, since just importing it was causing issues. (also, `process.env` is directly evaluated for ts libraries 😕)